src/magnify/apps/core/serializers/rooms.py

```diff
@@ -104,5 +104,6 @@ def to_representation(self, instance):
                     "groups": groups_serializer.data,
                 }
             )
+        output["has_admin_access"] = is_admin
```

Comment:
Why not keep the same name ?
```suggestion
        output["is_admin"] = is_admin
```

Reply:
My thinking was that `is_admin` is used on the `User` object to indicate that a given user is admin.
What we are indicating here is that the logged in user is admin OF the room on which we put this field... quite different right?

That's why I came out with a different name which, I agree, could be better but that's all I found :man_shrugging:  
Any idea with the context explained above?

Reply:
Oh, ok I think I get it : this is a room attribute, not a user one, right ?
If so, maybe something like
```suggestion
        output["administrable"] = is_admin
```

Reply:
I had the same feeling :sweat_smile: